zebra/zebra_rib.c

```diff
@@ -3228,7 +3228,7 @@ static wq_item_status meta_queue_process(struct work_queue *dummy, void *data)
 	/* Ensure there's room for more dataplane updates */
 	queue_limit = dplane_get_in_queue_limit();
 	queue_len = dplane_get_in_queue_len();
-	if (queue_len > queue_limit) {
+	if (queue_len > queue_limit || t_dplane) {
```

Comment:
In such a case debug message should be narrowed-down a bit to distinguish both cases.